doc/rst/source/datasets/remote-data.rst

```diff
@@ -0,0 +1,135 @@
+The first time you access one of these files, GMT will download the file (or a subset tile) from
+the selected GMT server and save it to the *server* directory under your GMT user directory [~/.gmt].
+From then on we read the local file from there.
+
+By using the remote file mechanism you should know that these files, on the server, will change
+from time to time (i.e., new versions are released, a problem in one file is fixed, or a dataset
+becomes obsolete), and GMT will take actions accordingly.  It is our policy to only supply the *latest*
+version of any dataset that undergoes revisions.  If you require previous versions for your work you
+will need to get those data from the data provider separately.  Unless you deactivate the remote data service,
+GMT will do the following when you request a remote file in a GMT command:
+
+#. We check if the locally cached catalog with information about the data available from the server
+   is up-to-date or if it needs to be refreshed.  If the file is older that the :term:`GMT_DATA_SERVER_UPDATE`
+   limit then we refresh the catalog.
+#. When the catalog is refreshed, we determine the publication date for each dataset on the server,
+   and if any local copies you may have are now obsolete will will remove them to force a re-download from the server.
+
+Usage
+~~~~~
+
+We have processed and reformatted publicly available global data sets (grids and images)
+and standardized their file names.  In GMT, you may access such data
+(or a subset only by using the **-R** option) by specifying the special name
+
+   @remote_name_\ *rr*\ *u*\ [_\ *reg*\ ]
+
+where the leading @ symbol identifies the file as a remote data set, the *remote_name_* is specific
+to the dataset and the *rr* code is a 2-digit integer specifying the grid/image
+resolution in the unit *u*, where *u* is either **d**, **m** or **s** for arc degree, arc minute or
+arc second, respectively. Optionally, you can append _\ **g** or _\ **p** to specifically get the
+gridline-registered or pixel-registered version (if they both exist).  If *reg* is not specified we
+will return the pixel-registered version unless only the gridline-registered file is available.  If you
+do specify a specific registration and that version is not available you will get an error message.
+The codes for *rr*\ *u* and the optional *reg* that are supported will be listed in the sections
+below describing each of the available data sets.
+
+Many of the remote datasets have a preferred, default color table that will be used unless you
+override that default by giving your desired CPT information.
+
+Controlling the Process
+~~~~~~~~~~~~~~~~~~~~~~~
+
+There are several ways you can control the remote data process and the amount of space taken up by your
+own server directory:
+
+#. You can select the GMT data server closest to you to minimize download time [:term:`GMT_DATA_SERVER`].
```

Comment:
Should also mention `GMT_AUTO_DOWNLOAD` here, but it may still have a bug (https://github.com/GenericMappingTools/gmt/issues/3608).

Reply:
OK, will add that

Reply:
Updated.